content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+---
+reviewers:
+- zparnold
+title: Overview of Cloud Native Security
+content_template: templates/concept
+weight: 1
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+Kubernetes Security (and security in general) is an immense topic that has many
+highly interrelated parts. In today's era where open source software is
+integrated into many of the systems that help web applications run,
+we need some overarching concepts to help guide our intuition about how we can
+think about security holistically. This guide will define a thought-framework for
+what we think of when we imagine "Kubernetes Security." The thought framework we selected is completely arbitrary
+and you should only use it if it helps you think about where to secure your software
+stack.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## The 4C's of Cloud Native Security
+Let's start with a diagram that may help you understand how we think about security in layers.
+{{< note >}}
+This layered approach augments the [defense in depth](https://en.wikipedia.org/wiki/Defense_in_depth_(computing))
+approach to security, which is widely regarded as a best practice for securing
+software systems. The 4C's are Cloud, Clusters, Containers, and Code.
+{{< /note >}}
+
+{{< figure src="/images/docs/4c.png" title="The 4C's of Cloud Native Security" >}}
+<br/><br/><br/>
+
+As you can see from the above figure of increasingly smaller areas of focus that
+each one of of the 4C's depend on the security of the squares in which they fit. It
+is nearly impossibly to safeguard against poor security standards in Cloud, Containers, and Code
```

Comment:
- do you need `increasingly smaller areas of focus`? The components are nested with levels of security increasing as you move to the outermost square?
- remove double `of`

Reply:
Yep, got it, thanks